source/dub/project.d

```diff
@@ -1396,6 +1396,15 @@ private string getVariable(Project, Package)(string name, in Project project, in
 
 	if (name == "BUILD_TYPE") return gsettings.buildType;
 
+	if (name == "DFLAGS" || name == "LFLAGS")
+	{
+		auto buildSettings = pack.getBuildSettings(gsettings.platform, gsettings.config);
+		if (name == "DFLAGS")
+			return join(cast(string[])buildSettings.dflags," ");
+		else if (name == "LFLAGS")
+			return join(cast(string[])buildSettings.lflags," ");
```

Comment:
Those casts weren't needed. I don't know why they were here in the first place, maybe because before another type was used.Anyways, I removed them.